src/main/const.ts

```diff
@@ -1,2 +1,23 @@
+import { APP_STORE_APP_ID, DEVHUB_APP_ID } from '$shared/const';
+import type { AppToInstall } from '$shared/types';
+
 export const WINDOW_SIZE = 600;
 export const SEARCH_HEIGH = 200;
+
+export const APP_STORE_SHA256 = 'e75e94b26e97e7ae9f8e9ea5d5ae0b532d561b7d12b4469275ea34ee06dcfd9c';
+export const DEVHUB_SHA256 = '30faeccb7c0333ffd5a01e3be111dac71773ad344ecb386f2f011bf61e513d96';
+
+export const APPS_TO_INSTALL: AppToInstall[] = [
+  {
+    id: APP_STORE_APP_ID,
+    sha256: APP_STORE_SHA256,
```

Comment:
Hard coding the sha256 here is potentially error prone if we replace the appstore/devhub happs.

Reply:
I'm fine with it for this PR but I think we should rather compute it based on the actual file.

Reply:
100% we should fix it along with not hardcoding binaries in fetch:default-apps